src/app-layer-ftp.c

```diff
@@ -32,7 +32,7 @@
 #include "app-layer-expectation.h"
 #include "app-layer-detect-proto.h"
 
-#include "rust.h"
+#include "rust-bindings.h"
```

Comment:
Why this change ?
I think we should always include rust.h, not rust-bindings...